server/models/schema.sql

```diff
@@ -449,7 +449,7 @@ CREATE TABLE debtor_group_subsidy (
 DROP TABLE IF EXISTS `depot`;
 CREATE TABLE `depot` (
   `uuid` BINARY(16) NOT NULL,
-  `text` VARCHAR(50) NOT NULL,
+  `text` VARCHAR(127) NOT NULL,
```

Comment:
I believe the max size we can have under `UTF8MB4` is VARCHAR(191).  Let's go ahead and make this VARCHAR(191) so that we don't run into this problem again.